civiform_config.example.sh

```diff
@@ -221,13 +221,19 @@ export APP_PREFIX="my-deploy" # max 19 chars, only numbers, letters, dashes, and
 # select correct region in web AWS console when creating certificate.
 export SSL_CERTIFICATE_ARN=""
 
-# RERUIRED
+# REQUIRED
 # Number of Civiform server tasks to run. This value can be set to 0 to shutdown servers.
 # It can be useful, for example, when server continiously fails on startup: set this to 0
 # to shutdown servers while figuring out the error.
 export FARGATE_DESIRED_TASK_COUNT=1
 
-
+# REQUIRED
+# The AWS RDS instance type for the Postgres database. For possible values, see:
+# https://github.com/civiform/cloud-deploy-infra/blob/main/cloud/aws/templates/aws_oidc/variable_definitions.json
+#
+# Changes to this value will result in database downtime, and should be applied during a
+# maintenance window.
+export POSTGRES_INSTANCE_CLASS="db.t3.micro"
```

Comment:
**suggestion:** for optional settings like this, commenting them out can indicate both the default value and that setting it is optional
```suggestion
# OPTIONAL
# The AWS RDS instance type for the Postgres database. For possible values, see:
# https://github.com/civiform/cloud-deploy-infra/blob/main/cloud/aws/templates/aws_oidc/variable_definitions.json
#
# Changes to this value will result in database downtime, and should be applied during a
# maintenance window.
# export POSTGRES_INSTANCE_CLASS="db.t3.micro"
```

Reply:
I'm not sure how to think about optional vs required here.

*Eventually*, the setting is of course required. But it's okay to not set it here, because we have code to set it to a default value. However, that seems subject to skew if we update the default in the future (update it in the code and forget to update the commented-out line here). For that reason it seems slightly safer to explicitly set the value here.

This is not a big deal to me. If the suggested change is conventional, it's fine with me.

Reply:
The audience for the comments in this file is civic entity IT staff for whom the rest of the deployment system is a black box. For their purposes, the setting is optional.